emcc.py

```diff
@@ -409,6 +409,9 @@ def standardize_setting_change(key, value):
     except Exception as e:
       exit_with_error('a problem occured in evaluating the content after a "-s", specifically "%s": %s', change, str(e))
 
+    if key == 'BINARYEN_TRAP_MODE':
+      exit_with_error('BINARYEN_TRAP_MODE is not supported by the LLVM wasm backend')
```

Comment:
maybe I'm missing something, but where is the check for the wasm backend? that is, the code looks like it will throw the error in fastcomp too?